api/src/Error/ErrorHandler.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Error;
+
+use ApiPlatform\GraphQl\Error\ErrorHandlerInterface;
+use Psr\Log\LoggerInterface;
+
+final class ErrorHandler implements ErrorHandlerInterface
+{
+    public function __construct(private ErrorHandlerInterface $defaultErrorHandler, private LoggerInterface $logger)
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __invoke(array $errors, callable $formatter): array
+    {
+
+        $this->logger->error('GraphQL errors', $errors);
```

Comment:
```suggestion
        $this->logger->critical($errors);
```

Il faudrait log en critical les erreurs >= 500, et sinon juste log un warning (sinon il va nous log les 404, etc... En critical)